libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -540,6 +540,7 @@ def write_mcu_config(f):
     else:
         f.write('#define HAL_USE_SDC FALSE\n')
         build_flags.append('USE_FATFS=no')
+        env_vars['DISABLE_SCRIPTING'] = True
```

Comment:
Why can't you just force the option bit?

Reply:
Well, for a start I don't know what that term means :-)

Reply:
`cfg.options.disable_scripting`

Reply:
I can't see that.  We seem to change env stuff based on config stuff - but not vice-versa in boards.py.  The script I'm setting the env var in doesn't have access to changing the config...

I'm going to merge this and someone suitably distraught by the changes can tell me off (for which I will then ask forgiveness (and detailed instructions on how to do it right) :-)